spec/system/candidate_completing_an_application_spec.rb

```diff
@@ -24,10 +24,25 @@
       click_on t('application_form.save_and_continue')
     end
 
-    it 'successfully submits the application' do
+    it 'can see the success page' do
       click_on t('application_form.submit')
 
       expect(page).to have_content t('application_form.application_submitted')
     end
+
+    it 'an email is sent' do
+      expect { click_on t('application_form.submit') }.to change { TTApplicationMailer.deliveries.count }.by(1)
+    end
```

Comment:
I think this spec is redundant, as the spec below would fail if no email were sent

Reply:
If we decide to keep this spec I think we should rename it to `can send an email of the submitted application` or something along those lines.